testdata/p4_16_samples_outputs/xor_test-frontend.p4

```diff
@@ -67,12 +67,12 @@ control MyIngress(inout headers hdr, inout metadata meta, inout standard_metadat
     }
     @name("MyIngress.forward_and_do_something") action forward_and_do_something(@name("port") egressSpec_t port) {
         standard_metadata.egress_spec = port;
-        if (hdr.ipv4.isValid()) {
+        if (hdr.ipv4.isValid()) @inlinedFrom("srcAddr_0") {
```

Comment:
Shouldn't this be `srcAddr`? `srcAddr_0` is the temporary name which you can't trace back to the original program.

Reply:
This might require the use of `originalName`, if it is well-preserved.

Reply:
Oh, right. Fixed!